README.md

```diff
@@ -20,12 +20,14 @@ In this "choose your own adventure" template portfolio, we have a [Blazor](https
 ### Quick Start
 
 1. Click the **Use this Template** button
+   
+   [![Use this Template](/images/template-button.svg)](https://github.com/education/codespaces-project-template-dotnet/generate)
 2. Select the repository owner (e.g. your GitHub account)
 3. Enter a unique name for your new repository
 4. Click the **Code** button
 5. Click **Create Codespace on main** button
-1. [Customize your portfolio site](#-customize-your-site-in-4-steps)
-1. [Deploy your site](#-deploy-your-web-application)
+6. [Customize your portfolio site](#-customize-your-site-in-4-steps)
```

Comment:
you can redo all the numbering to be `-1.` and the markdown engine will numerate them correctly

Reply:
Changed!